README.md

```diff
@@ -122,9 +122,16 @@ You are able to execute command functions without any argument validation. If yo
 import webExt from 'web-ext';
 
 webExt.cmd.run({
+  // To run an extension on android, uncomment next three lines
```

Comment:
> @rpl I have tried to update the documentation on `how to run an extension on android` as we talked about on addons matrix channel, but I am not really happy with adding all those comments. Do you have any suggestions on how to write that?

@ankushduacodes How about moving everything related to android (this commented fragments and the new parts describing the two adb utility methods) in a "single paragraph + code snipped" at the end of this section?

I think that something like the following may be enough for now:

```
If you would like to run an extension on Firefox for Android:

// Path to adb binary (optional parameter, auto-detected if missing)
const adbBin = ...
// Get an array of device ids (Array<string>)
const deviceIds = await webExt.util.adb.listADBDevices(adbBin);
const adbDevice = ...
// Get an array of Firefox APKs (Array<string>)
const firefoxAPKs = await webExt.util.adb.listADBFirefoxAPKs(
  deviceId, adbBin
);
const firefoxApk = ...

webExt.cmd.run({
  target: 'firefox-android',
  firefoxApk,
  adbDevice,
  sourceDir: ...
}).then((extensionRunner) => ...);
```

How that sounds to you?

Reply:
Using await means we will have to add an async function to make it work, no?

Reply:
@ankushduacodes if I recall correctly nodejs will support top level `await` starting from nodejs 14, anyway I wouldn't bother to explicitly mention that here (I guess that who is using web-ext as a library is going to be a developer, and we may just assume that they will know how to use `await` in the nodejs version they are targeting).

Reply:
@rpl I have updated the docs, Please have a look and let me know there needs to be any change.

Also, are two new tests looking alright?